pyramid_nacl_session/utils.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from binascii import unhexlify
+
+from pyramid.session import JSONSerializer
+from pyramid.session import PickleSerializer
+
+from pyramid.settings import asbool
+
+from .session import EncryptedCookieSessionFactory
+
+
+def session_factory_from_settings(settings):
+    """ Return a Pyramid session factory using PyNaCl session settings
+    supplied from a Paste configuration file"""
```

Comment:
Nitpick: I prefer multi-line docstrings to have `"""` on their own separate lines. Nonetheless, docstrings must follow English punctuation rules.

```
    """
    Return a Pyramid session factory using PyNaCl session settings
    supplied from a Paste configuration file.
    """
```